provision/minikube/keycloak/templates/keycloak-initial-admin-secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
This file looks unused in this change, am I missing something?

Reply:
If I would not add this secret, the keycloak operator would create a secret with a random password for the admin, and this would be annoying. Pre-creating this secret allows me to have a pre-defined well-known password for the admin user in this setup. 

Therefore it's related and I'd like to keep it.

Reply:
You are right, thanks for explaining, not sure if this is a bug or a feature though 😄 
https://github.com/keycloak/keycloak/blob/c31d37ddf1c3e3529e761800a6842fdbf25ba931/operator/app/src/main/java/org/keycloak/operator/controllers/KeycloakAdminSecret.java#L24

cc. @vmuzikar

Reply:
Undocumented feature. 🙂